app/src/test/java/com/codesoom/assignment/utils/JwtUtilTest.java

```diff
@@ -0,0 +1,67 @@
+package com.codesoom.assignment.utils;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.security.SignatureException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@Nested
+@DisplayName("JwtUtil 테스트")
+class JwtUtilTest {
+    private static final String SECRET = "01234567890123456789012345678901";
+    private static final String VALID_TOKEN = "eyJhbGciOiJIUzI1NiJ9" +
+            ".eyJ1c2VySWQiOjF9._sqAnLqnuii5tTri0u8AAwGJpI4PF6WRT9wkOLyxWaw";
+    private static final String INVALID_TOKEN = "eyJhbGciOiJIUzI1NiJ9" +
+            ".eyJ1c2VySWQiOjF9._sqAnLqnuii5tTri0u8AAwGJpI4PF6WRT9wkOLyxWa0";
```

Comment:
INVALID 토큰을 이렇게 만들어 주는 것도 방법일 거에요.
```suggestion
    private static final String INVALID_TOKEN = VALID_TOKEN + "0";
```